src/classes/ERR_Handler.cls

```diff
@@ -239,10 +332,25 @@ public class ERR_Handler {
             error.Error_Type__c = getStatusCode(undeleteResult.getErrors()[0].getStatusCode());
             error.Full_Message__c = undeleteResult.getErrors()[0].getMessage();
     	}
-    
+
         UTIL_Debug.debug('**** createError() returns error: ' + error);
         return error;
     }
+
+	/*******************************************************************************************************
+	* @description Creates a single error record from a Database class DML operation in a batch, future or queued job.
+	* @param message The full message of all the errors for a given job and compiled IDs.
+	* @return The error record.
+	*/
+	private static Error__c createJobError(String message) {
+		Error__c error = new Error__c();
+		error.Datetime__c = System.now();
+		error.Full_Message__c = message;
+		error.Error_Type__c = 'Job Error';
```

Comment:
Do you think there's any value including parameters or method overloads to pass in values for the Context__c and/or Object__c fields?

Reply:
Do we have any docs on how the context__c field should be populated? I like the overload for the object__c, but would need to determine how best to pass that information.

Reply:
There's a Enum with context values, though in my opinion it wouldn't hurt to refactor that at some point.